libcloud/ovh.cfg.orig

```diff
@@ -70,12 +70,14 @@ libcloud:
             - systemctl restart ntpd.service
```

Comment:
Having a commit message that explains why we want to switch to v3 would be useful imo (sorry for commenting here - github does not allow me to comment on the commit message)

Reply:
no idea why do you need this, in a commit message, it's trivial
btw, this PR I gave for review just in education reasons :-) is not supposed to make to much explanation, so I want it merge asap

Reply:
because a commit message is the first thing I look at when I try to find out when/why somebody changed something (via `git blame`).
And it's not trivial. Let me ask you in 1 year - I'm sure you'll not remember that OVH ended v2 support and that's the reason why you changed it. anyway - feel free to merge.

Reply:
ok, usually I do and add more descriptive explanation, but for this particular case it is absolutely not important why to switch from my point of view. Also, keep in mind that this is just a template, where are examples given of all possible/working configs for us at least, so the real switch will happen only after we update secret file with updated credentials on jenkins server ;-)

Reply:
in 1 year after we switched all our instances to auth v3 there will not be a ha why we switched digging in git log, so I see